2.6.md

```diff
@@ -972,13 +998,24 @@ Recommended for video and/or apps. </td>
     <td>integer</td>
     <td>Advisory as to the number of seconds that may elapse between the auction and the actual impression.</td>
   </tr>
+  <tr>
+    <td><code>qty</code></td>
+    <td>object</td>
+    <td>A means of passing a multiplier in the bid request, representing the total quantity of impressions for adverts that display to more than one person.</td>
+  </tr>
+  <tr>
+    <td><code>dt</code></td>
+    <td>float</td>
```

Comment:
IMO this should be an integer and only needs to be seconds since the epoch. If this has been discussed and resolved already, please indicate that here.